### PR TITLE
[WIP] integration with w3c VC

### PR DESCRIPTION
**Pull request by ilya-korotya**
**!** Temporary path to `iden3credential-v2.json-ld`:
https://gist.githubusercontent.com/ilya-korotya/049b77417f14964235ffc864c07e3d66/raw/bd2f2d5b348e1e77b9ad34b7255a193e2fcf64ee/iden3credential.json-ld </br >
Example of VC from identity server: (need rename `expiration` to `expirationDate`):
```json
{
    "id": "91e31521-9bce-4f59-bac3-09b3565de8c4",
    "@context": [
      	"https://www.w3.org/2018/credentials/v1",
        "https://gist.githubusercontent.com/ilya-korotya/049b77417f14964235ffc864c07e3d66/raw/bd2f2d5b348e1e77b9ad34b7255a193e2fcf64ee/iden3credential.json-ld",
        "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v2.json-ld"
    ],
    "@type": [
      "VerifiableCredential", "Iden3Credential"
    ],
    "expirationDate": "2361-03-21T21:14:48+02:00",
    "updatable": false,
    "version": 0,
    "rev_nonce": 974451574,
    "credentialSubject": {
        "birthday": 19960424,
        "documentType": 1,
        "id": "118yLweameMMCqCLU9UyHJGoCsdKLrM9NxBRiR1aLX",
        "type": "KYCAgeCredential"
    },
    "credentialStatus": {
        "id": "http://localhost:8010/node",
        "type": "Iden3SparseMerkleProof"
    },
    "subject_position": "index",
    "credentialSchema": {
        "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v2.json-ld",
        "type": "Iden3JSONLDValidator"
    },
    "proof": [
        {
            "@type": "BJJSignature2021",
            "issuer_data": {
                "id": "117BhsDYGUtXccymngi7HuMv4uLiLGjcEiJsLWfLfk",
                "state": {
                    "claims_tree_root": "7f4aba6eb35a0a228661eaa9e5e5de536cdf8a22a95c5cdf472ff87cf9ac3823",
                    "value": "49ee5b26f8883b9824b00a937f506b6cb55c5f5c480cf0d33b64f27779f04706"
                },
                "auth_claim": [
                    "304427537360709784173770334266246861770",
                    "0",
                    "15400654713677258494252118126906640056902074141967165198835953550898955246038",
                    "14748337183994671759430923533746918452739238335738545884653591536771871780403",
                    "0",
                    "0",
                    "0",
                    "0"
                ],
                "mtp": {
                    "existence": true,
                    "siblings": []
                },
                "revocation_status": "http://localhost:8001/api/v1/identities/117BhsDYGUtXccymngi7HuMv4uLiLGjcEiJsLWfLfk/claims/revocation/status/0"
            },
            "signature": "c48b9da6147febab8b985925ff3826c98606a417e64d2f1b3edbdb4258fc1b2f7e8cdbdfec5553d07ee7e9679eb97fff523da85eee5add6582417d6b7c471c03"
        }
    ]
}
```
Example of resolve VC has been issued identity server:
```json
[
  {
    "@type": [
      "https://www.w3.org/2018/credentials#VerifiableCredential",
      "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential.json-ld#Iden3Credential"
    ],
    "https://www.w3.org/2018/credentials#credentialSchema": [
      {
        "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v2.json-ld",
        "@type": [
          "Iden3JSONLDValidator"
        ]
      }
    ],
    "https://www.w3.org/2018/credentials#credentialStatus": [
      {
        "@id": "http://localhost:8010/node",
        "@type": [
          "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential.json-ld#Iden3SparseMerkleProof"
        ]
      }
    ],
    "https://www.w3.org/2018/credentials#credentialSubject": [
      {
        "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/kyc.md#birthday": [
          {
            "@type": "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/serialization.md#IndexDataSlotA",
            "@value": 19960424
          }
        ],
        "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/kyc.md#documentType": [
          {
            "@type": "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/serialization.md#IndexDataSlotB",
            "@value": 1
          }
        ],
        "@id": "118yLweameMMCqCLU9UyHJGoCsdKLrM9NxBRiR1aLX",
        "@type": [
          "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc.json-ld#KYCAgeCredential"
        ]
      }
    ],
    "https://www.w3.org/2018/credentials#expirationDate": [
      {
        "@type": "http://www.w3.org/2001/XMLSchema#dateTime",
        "@value": "2361-03-21T21:14:48+02:00"
      }
    ],
    "@id": "91e31521-9bce-4f59-bac3-09b3565de8c4",
    "https://w3id.org/security#proof": [
      {
        "@graph": [
          {
            "@type": [
              "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential.json-ld#BJJSignature2021"
            ],
            "https://github.com/iden3/claim-schema-vocab/blob/main/proofs/BJJSignature2021.md#issuer_data": [
              {
                "@id": "117BhsDYGUtXccymngi7HuMv4uLiLGjcEiJsLWfLfk"
              }
            ],
            "https://github.com/iden3/claim-schema-vocab/blob/main/proofs/BJJSignature2021.md#signature": [
              {
                "@type": "https://w3id.org/security#multibase",
                "@value": "c48b9da6147febab8b985925ff3826c98606a417e64d2f1b3edbdb4258fc1b2f7e8cdbdfec5553d07ee7e9679eb97fff523da85eee5add6582417d6b7c471c03"
              }
            ]
          }
        ]
      }
    ],
    "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/common.md#rev_nonce": [
      {
        "@type": "http://www.w3.org/2001/XMLSchema#integer",
        "@value": 974451574
      }
    ],
    "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/common.md#updatable": [
      {
        "@type": "http://www.w3.org/2001/XMLSchema#boolean",
        "@value": false
      }
    ],
    "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/common.md#version": [
      {
        "@type": "http://www.w3.org/2001/XMLSchema#integer",
        "@value": 0
      }
    ]
  }
]
```

### CODE REVIEW

File: schemas/json-ld/iden3credential-v2.json-ld [hunk 1]
```diff
@@ -0,0 +1,230 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+    "Iden3Credential": {
+      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential.json-ld#Iden3Credential",
```
Comment (vmidyllic) [inline schemas/json-ld/iden3credential-v2.json-ld:8]:
```suggestion
      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential-v2.json-ld#Iden3Credential",
```

File: schemas/json-ld/iden3credential-v2.json-ld [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        },
+        "subjectPosition": {
+          "@id": "iden3-vocab:subjectPosition",
+          "@type": "xsd:string"
+        },
+        "Iden3JSONLDValidator": "cred:Iden3JSONLDValidator"
+      }
+    },
+    "Iden3SparseMerkleProof": {
+      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential.json-ld#Iden3SparseMerkleProof",
```
Comment (vmidyllic) [inline schemas/json-ld/iden3credential-v2.json-ld:39]:
```suggestion
      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential-v2.json-ld#Iden3SparseMerkleProof",
```

File: schemas/json-ld/iden3credential-v2.json-ld [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+        },
+        "h_value": {
+          "@id": "smt-proof-vocab:h_value",
+          "@nest": "node_aux",
+          "@type": "xsd:string"
+        }
+      }
+    },
+    "BJJSignature2021": {
+      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential.json-ld#BJJSignature2021",
```
Comment (vmidyllic) [inline schemas/json-ld/iden3credential-v2.json-ld:124]:
```suggestion
      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential-v2.json-ld#BJJSignature2021",
```

File: schemas/json-ld/iden3credential-v2.json-ld (hunk 1 above)
Comment (ilya-korotya) [reply schemas/json-ld/iden3credential-v2.json-ld]: done

File: schemas/json-ld/iden3credential-v2.json-ld (hunk 2 above)
Comment (ilya-korotya) [reply schemas/json-ld/iden3credential-v2.json-ld]: done

File: schemas/json-ld/iden3credential-v2.json-ld (hunk 3 above)
Comment (ilya-korotya) [reply schemas/json-ld/iden3credential-v2.json-ld]: done

**Review by vmidyllic — DISMISSED** [review]

**Review by vmidyllic — APPROVED** [review]

**Pull request merged**